geowave-accumulo/src/main/java/mil/nga/giat/geowave/accumulo/util/StatsTool.java

```diff
@@ -75,6 +77,12 @@ public static boolean calculateStastics(
 			}
 		}
 		statsTool.flush();
+		try {
+			statsTool.close();
```

Comment:
true...just left it because the tool is hopefully temporary to something more robust and the close is the last step of the process.